docs/running.rst

```diff
@@ -33,8 +33,48 @@ To setup the server locally:
     $ mkdir ddb
     $ curl -sSL http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest.tar.gz | tar xzvC ddb/
     $ java -Djava.library.path=./ddb/DynamoDBLocal_lib -jar ./ddb/DynamoDBLocal.jar -sharedDb -inMemory
-    $ cp automock/boto.cfg ~/.boto
 
-Note the last line copies a boto config over ``~/.boto`` in your home dir. If
-you have existing AWS Credentials in this file, you should move it elsewhere
-first before running ``autopush`` and ``autoendpoint``.
+An example `boto config file`_ is provided in ``automock/boto.cfg`` that
+directs autopush to your local DynamoDB instance.
+
+Configuring the Nodes
+=====================
+
+Besides DynamoDB, at a bare minimum, autopush requires configuration of a token
+encryption key generated by the ``autokey`` program. This key should be shared
+by Connection and Endpoint nodes in the same cluster.
+
+To generate and configure a key:
+
+.. code-block:: bash
+
+    $ bin/autokey
+    Key = <crypto key here>
+    $ cp configs/autopush_shared.ini .
+
+Then edit your ``autopush_shared.ini`` to include it:
+
+.. code-block:: ini
+
+    crypto_key = [<paste crypto key here>]
+
+You may also want to enable human readable logs in the shared config file:
+
+.. code-block:: ini
+
+    ; Uncomment to provide stdout logging in a more human readable format.
+    #human_logs
+
+Running the nodes
+=================
+
+.. code-block:: bash
+
+    $ BOTO_CONFIG=automock/boto.cfg bin/autopush --config-shared=./autopush_shared.ini
```

Comment:
hrm. configs are weird. 

We have an app specific file, plus the shared. We also default to reasonably well known locations. It seems odd that we're using the CLI to specify just one config, but I agree that specifying both could be verbose and potentially confusing. I'll file a bug to consider simplifying this by adding a `config_dir` arg or something.